components/coding/GoalsSectionComponent.tsx

```diff
@@ -16,17 +23,30 @@ export default function GoalsSection({
   return (
     <div>
       {userGoals.length > 0 && (
-        <div className="grid grid-cols-5 text-sm font-semibold text-center border-b-2 md:grid-cols-10 md:text-lg">
+        <div className="grid grid-cols-5 text-sm font-semibold text-center border-b-2 md:grid-cols-12 md:text-lg">
           <p className="font-semibold">{sectionName}</p>
           <p className="col-span-2 md:col-span-4">Goal</p>
           <p className="hidden md:block md:col-span-2">Date Added</p>
           <p className="font-semibold md:col-span-2">Target Date</p>
+          <p className="hidden md:block md:col-span-2">Days Remaining</p>
         </div>
       )}
 
       {userGoals.map((goal, index) => {
         return (
-          <div className="grid grid-cols-5 my-2 text-sm text-center md:grid-cols-10 md:text-lg place-items-center">
+          <div
```

Comment:
logic is perfect but I think we can clean this up by defining the logic outside the component so our JSX is cleaner... maybe something like goalStyle = {differenceInCalendarDays(...)}, then reference goalStyle inside className

Reply:
did that. thank you!